WPMedia/arc-themes-components
# TMEDIA-688 - Icon Component

Added the Icon Component and initial set of Icons as per acceptance criteria

### Ticket 
[TMEDIA-688](https://arcpublishing.atlassian.net/browse/TMEDIA-688)

### Testing
* Checkout branch - `git checkout TMEDIA-688-icon-component`
* Run storybook - `npm run storybook`
* Check storybook output examples and documentation
* Check chromatic build
